### PR TITLE
multipledispatch: init at 0.4.8

### PR DESCRIPTION
**Pull request by FRidh**

### CODE REVIEW

**Comment by FRidh** [comment]
> No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.
> The build has been terminated

Package builds.

Couldn't import though, see #8635

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -7574,6 +7574,21 @@ let
...
+    version = "0.4.8";
+   
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/multipledispatch/${name}.tar.gz";
+      md5 = "f0eb0c527cdbe788ca7dcd1511bc49a9";
+    };
+    
+    meta = {
+      homepage = "http://github.com/mrocklin/multipledispatch/";
+      description = "A relatively sane approach to multiple dispatch in Python.";
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: Descriptions should not end with a dot. See http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -7574,6 +7574,21 @@ let
...
+    name = "multipledispatch-${version}";
+    version = "0.4.8";
+   
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/multipledispatch/${name}.tar.gz";
+      md5 = "f0eb0c527cdbe788ca7dcd1511bc49a9";
+    };
+    
+    meta = {
+      homepage = "http://github.com/mrocklin/multipledispatch/";
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: You do not need quotes around urls.

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -7574,6 +7574,21 @@ let
     propagatedBuildInputs = with self; [ eventlib application gnutls ];
   };
 
+  multipledispatch = buildPythonPackage rec {
+    name = "multipledispatch-${version}";
+    version = "0.4.8";
+   
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/multipledispatch/${name}.tar.gz";
+      md5 = "f0eb0c527cdbe788ca7dcd1511bc49a9";
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: Despite there are quite a few packages using md5 sums I think it is better to use sha256.

**Comment by lethalman** [comment]
I'm able to import it. Not sure what are you doing wrong with nix-shell :)

**Pull request merged**